staging/src/k8s.io/kube-scheduler/OWNERS

```diff
@@ -0,0 +1,12 @@
+approvers:
+- api-approvers
+- sig-scheduling-maintainers
+- dixudx
```

Comment:
drop from approver here, leaning on api-approvers and sig-scheduling-maintainers alias